azure-pipelines.yml

```diff
@@ -1,38 +1,41 @@
 trigger:
-- master
+- updates/strategy-tweak
```

Comment:
This should not go to master...

Reply:
Or at least, not with this specific branch. It should be at least a wildcard pattern, e.g. for topics/* or something like that.

Reply:
I have reverted this trigger back to master